components/Fundraise/api/contributions.ts

```diff
@@ -0,0 +1,76 @@
+import { Helpers } from "@quantfive/js-web-config";
+import API, { generateApiUrl } from "~/config/api";
+import { ID } from "~/config/types/root_types";
+import { captureEvent } from "~/config/utils/events";
+import { isNullOrUndefined } from "~/config/utils/nullchecks";
+import { Purchase, parsePurchase } from "~/config/types/purchase";
+import { Fundraise, parseFundraise } from "../lib/types";
+
+export const fetchFundraiseContributions = async ({
```

Comment:
To make it super easy to understand whether a function is utility vs api call, we suffix API functions with `Api` suffix. It reads a little funny but it gets the job done. Could you update the API functions here:
e.g. `fetchFundraiseContributionsApi`